tests/mapping/test_BaseMappingTransformer.py

```diff
@@ -110,7 +115,13 @@ def test_mappings_unchanged(self, uninitialized_transformers):
             "b": {"a": 1, "b": 2, "c": 3, "d": 4, "e": 5, "f": 6},
         }
 
-        x = uninitialized_transformers[self.transformer_name](mappings=mapping)
+        if self.transformer_name == "CrossColumnMappingTransformer":
```

Comment:
Not sure I like having conditional splits in logic for different transformers within a test like this.  Might be clearer and less coupled if specific behaviour for children is specified and tested in its own test module. Can we just overwrite the test for CrossColumnMappingTransformer? Appreciate that means duplicating code but would like to keep the logical flow of these tests consentient.

Reply:
ah overwriting/overloading makes sense thanks, updated [here](https://github.com/lvgig/tubular/pull/236/commits/bbffa2b5b196e12d92565f0500c3f81c79479b0d)